eduNEXT/edunext-platform
# translations: downloaded lilac translations

**description:**
Updated translations and downloaded from transifex.

**changes:**
updated i18n and static files from:
- lms/static
- cms/static
updated locales from:
- conf/locale

**how to test:**
check the pages and change the translations with the language selector into header.

**jira:**
https://edunext.atlassian.net/browse/PS2021-1374
